local builds issue with libaom

Hello,

I'm trying to make a local nightly build of AROS.
That always fail with `cannot find -laom`.

I'm using those command lines from the nightly build logs:

```log
configure --target=pc-x86_64 --enable-target-variant=smp \
--with-gcc-version=15.2.0 --enable-ccache --enable-build-type=nightly \
--with-portssources=/a/src/AROS/bin/Sources \
--with-aros-toolchain-install=/a/x-tools/x86_64-aros
```
... idem with

```log
configure --target=linux-x86_64 --with-gcc-version=13.4.0 --enable-ccache \
--enable-unit-tests=no --enable-build-type=nightly \
--with-portssources=/a/src/AROS/bin/Sources \
--with-aros-toolchain-install=/a/x-tools/aros/linux-x86_64-13.4.0

MMAKE] Making workbench-datatypes-html-fd in workbench/classes/datatypes/html
Generating AROS/Developer/SDK/fd/html_lib.fd
[MMAKE] Making workbench-datatypes-pnm-fd in workbench/classes/datatypes/pnm
Generating AROS/Developer/SDK/fd/pnm_lib.fd
make[2]: *** No targets specified and no makefile found. Stop.
make[2]: *** No targets specified and no makefile found. Stop.
[MMAKE] Making workbench-datatypes-heic in workbench/classes/datatypes/heic
Makedepend workbench/classes/datatypes/heic/heicdummy.cc ...
Generating support files for module heic
Makedepend bin/linux-x86_64/gen/workbench/classes/datatypes/heic/heic/heic_end.c ...
Makedepend bin/linux-x86_64/gen/workbench/classes/datatypes/heic/heic/heic_start.c ...
Makedepend workbench/classes/datatypes/heic/heicclass.c ...
Compiling bin/linux-x86_64/gen/workbench/classes/datatypes/heic/heic/heic_start.c 
Compiling workbench/classes/datatypes/heic/heicclass.c 
Compiling workbench/classes/datatypes/heic/heicdummy.cc 
Compiling bin/linux-x86_64/gen/workbench/classes/datatypes/heic/heic/heic_end.c 
Building Module AROS/Classes/DataTypes/heic.datatype ...
ccache /a/x-tools/aros/linux-x86_64-13.4.0/x86_64-aros-g++ -m64 -mcmodel=large -mno-red-zone -mno-ms-bitfields -nostartfiles -Wl,-Map -Xlinker /a/src/AROS-linux-x86_64-13.4.0/bin/linux-x86_64/gen/workbench/classes/datatypes/heic/heic/.map /a/src/AROS-linux-x86_64-13.4.0/bin/linux-x86_64/gen/compiler/libinit/libentry.o /a/src/AROS-linux-x86_64-13.4.0/bin/linux-x86_64/gen/workbench/classes/datatypes/heic/heic/heic_start.o /a/src/AROS-linux-x86_64-13.4.0/bin/linux-x86_64/gen/workbench/classes/datatypes/heic/heic/heicclass.o /a/src/AROS-linux-x86_64-13.4.0/bin/linux-x86_64/gen/workbench/classes/datatypes/heic/heic/heicdummy.o /a/src/AROS-linux-x86_64-13.4.0/bin/linux-x86_64/gen/workbench/classes/datatypes/heic/heic/heic_end.o -o /a/src/AROS-linux-x86_64-13.4.0/bin/linux-x86_64/AROS/Classes/DataTypes/heic.datatype -static-libstdc++ -L/a/src/AROS-linux-x86_64-13.4.0/bin/linux-x86_64/gen/workbench/classes/datatypes/heic/lib -Wl,--start-group -lheif -laom -lde265 -lposixc -lstdc -latomic -Wl,--end-group
/a/x-tools/aros/linux-x86_64-13.4.0/x86_64-aros-ld: cannot find -laom
make[1]: *** [mmakefile:3481: /a/src/AROS-linux-x86_64-13.4.0/bin/linux-x86_64/AROS/Classes/DataTypes/heic.datatype] Error 1
[MMAKE] make --no-print-directory TOP=/a/src/AROS-linux-x86_64-13.4.0 SRCDIR=/a/src/AROS CURDIR=workbench/classes/datatypes/heic TARGET=workbench-datatypes-heic --file=mmakefile workbench-datatypes-heic failed: 512
[MMAKE] Error: Error while running make in workbench/classes/datatypes/heic: No such file or directory
make: *** [Makefile:128: all] Error 10
```

As those nightly builds are okay according to the github page, tell me how can I fix that.

Thanks,

PS: trying with --enable-unit-tests=yes fails on some cunit header file not found.
PS: I added -v to mmake args in Makefile
PS: this is about commit b7c005a522, HEAD as of today, the nighly build logs should contain that info

--- a/config/make.tmpl
+++ b/config/make.tmpl
@@ -4103,7 +4103,7 @@ ifneq (%(aros_prefix),)
 else
     %(mmake)-aros_prefix     := $(%(mmake)-prefix)
 endif
-
+    %(mmake)-cmake-env += CMAKE_GENERATOR='Unix Makefiles'
 ifeq (%(compiler),host)
     %(mmake)-cmake-env += LDFLAGS="$(HOST_LDFLAGS) -L$(CROSSTOOLSDIR)/lib $(USER_LDFLAGS) %(extraldflags)"
     %(mmake)-cmake-env += CFLAGS="$(HOST_CFLAGS) -I$(CROSSTOOLSDIR)/include %(extracflags)"

